Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -204,14 +204,14 @@ protected async Task OnTextChangedHandler( string text )
                 }
             }
 
-            if ( NotFound.HasDelegate && !HaveFilteredData )
+            if ( HaveFilteredData )
```

Comment:
I too was wondering about it, and I ended up asking "have I filtered data?", instead of 
"has component filtered data?" - maybe "has" is more recognizable as a boolean, i'll change it.

Reply:
Ye, I pretty much only seen `Has` for these type of booleans, and also use it myself.  And you do have a Blazor HasDelegate just behind it haha.